Resources/Prototypes/Entities/Structures/Furniture/rollerbeds.yml

```diff
@@ -71,6 +71,7 @@
             False: {visible: true}
     - type: StaticPrice
       price: 120
+    - type: DeployFoldable
```

Comment:
could you also add this to the body bag? It seems like a natural thing that would share this behavior.

Reply:
Done. Also added to folding chairs.